tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -118,6 +118,46 @@ When(
   }
 )
 
+Given(
+  'user {string} from server {string} has shared {string} with user {string} from remote/local server',
```

Comment:
`remote/local` has no effect so I could write
`Given user "artur" from server "local" has shared "lorem.txt" with user "peter@very-far-away-server" from local server`

Reply:
I didn't get it. Could you explain to me more?